ReSharper.FSharp/src/FSharp.Psi/src/IFSharpTypeMember.cs

```diff
@@ -39,4 +39,8 @@ public interface IFSharpGeneratedFromOtherElement : IFSharpGeneratedElement
   public interface IFSharpGeneratedFromUnionCase : IFSharpGeneratedFromOtherElement
   {
   }
+
+  public interface IFSharpGeneratedFromProperty : IFSharpGeneratedFromOtherElement
```

Comment:
Was this interface introduced for a single implementation only? What other types you expect there?
Would it be better to add an interface like `IFSharpGetterInstead`?

Reply:
Another implementation that is not in this PR is the property-indexer

Reply:
What about ```IFSharpGeneratedAccessor```?

Reply:
Right, it'd be better.